src/gestures/PanSession.ts

```diff
@@ -1,12 +1,12 @@
-import { EventInfo, Point } from "../events/types"
+import { EventInfo } from "../events/types"
 import { isTouchEvent, isMouseEvent } from "./utils/event-type"
 import { extractEventInfo } from "../events/event-info"
 import sync, { getFrameData, cancelSync } from "framesync"
 import { secondsToMilliseconds } from "../utils/time-conversion"
 import { addPointerEvent } from "../events/use-pointer-event"
 import { unblockViewportScroll } from "../gestures/drag/utils/block-viewport-scroll"
 import { distance } from "popmotion"
-import { TransformPoint2D } from "../types/geometry"
+import { Point2D, TransformPoint2D } from "../types/geometry"
```

Comment:
This can be imported as types:
```suggestion
import type { Point2D, TransformPoint2D } from "../types/geometry"
```